app/src/components/widgets/charts/investigatedTrendChart/common/utils.js

```diff
@@ -0,0 +1,24 @@
+import * as COLORS from 'common/constants/colors';
+import { range as getRange } from 'components/widgets/charts/common/utils';
+
+export const getLaunchAxisTicks = (itemsLength) => {
```

Comment:
I see the same function here: `app/src/components/widgets/charts/common/utils/utils.js`

Reply:
fixed